Source/Models/SwishError.swift

```diff
@@ -3,7 +3,8 @@ import Argo
 
 public enum SwishError {
   case ArgoError(Argo.DecodeError)
-  case InvalidJSONResponse(NSError)
+  case InvalidDeserializeError(NSError)
+  case InvalidParseError(NSError)
```

Comment:
I think someone did mention at one point to remove Error from the end of them all. I'd be in support of that.